scripts/sct_compute_snr.py

```diff
@@ -55,20 +56,20 @@ def get_parser():
         help="show this help message and exit")
     optional.add_argument(
         '-m',
-        help='Binary (or weighted) mask within which SNR will be averaged (e.g. "dwi_moco_mean_seg.nii.gz").',
+        help='Binary (or weighted) mask within which SNR will be averaged. Example: dwi_moco_mean_seg.nii.gz',
         metavar=Metavar.file,
         default='')
     optional.add_argument(
         '-method',
-        help='Method to use to compute the SNR:\n'
+        help='R|Method to use to compute the SNR:\n'
              '- diff (default): Substract two volumes (defined by -vol) and estimate noise variance within the ROI (flag -m is required).\n'
```

Comment:
could you replace the "-" with " ". (people might write "-")

Reply:
Sure! Added in commit [6aeb39](https://github.com/neuropoly/spinalcordtoolbox/commit/6aeb39bdaac52041c2d295c1ffacb0e9961df50a)